src/main/scala/io/iohk/ethereum/db/ReceiptStorage.scala

```diff
@@ -0,0 +1,17 @@
+package io.iohk.ethereum.db
+
+import io.iohk.ethereum.network.p2p.messages.PV63.Receipt
+import io.iohk.ethereum.rlp.RLPImplicits._
+import io.iohk.ethereum.rlp.{decode => rlpDecode, encode => rlpEncode}
+
+object ReceiptStorage {
+  type BlockHash = Array[Byte]
+  type ReceiptStorage = KeyValueStorage[BlockHash, Seq[Receipt]]
+
+  def apply(dataSource: DataSource): ReceiptStorage = new ReceiptStorage(
+    dataSource = dataSource,
+    keySerializer = identity,
+    valueSerializer = rlpEncode(_)(seqEncDec[Receipt]),
+    valueDeserializer = rlpDecode[Seq[Receipt]](_)(seqEncDec[Receipt])
+  )
```

Comment:
A `put` method similar to the one implemented in `NodeStorage`? The problem is that in that case we can't derive the `BlockHash` from the `Seq[Receipt]`. There is also a method `def put(key: K, value: V)` defined in the `KeyValueStorage` class that can be used in that case.